src/configuration/incentives.ts

```diff
@@ -16,6 +16,10 @@ export const incentivesConfigurationSchema = T.Object({
    * The encrypted key to use for permit generation
    */
   evmPrivateEncrypted: T.String(),
+  /**
+   * Reward token for ERC20 permits
+   */
+  erc20RewardToken: T.String(),
```

Comment:
Shall we consider defaulting it to have `WXDAI`?

Reply:
Updated https://github.com/ubiquibot/conversation-rewards/pull/40/commits/f7ad97538c7a5da1dfee37f309be4a2885847574

QA: [config](https://github.com/rndquu-org/test-repo/blob/bfc5252b5cb4ec4cf7125f528ca7eeab16ef0a48/.github/.ubiquibot-config.yml) without the `erc20RewardToken` param and generated [permit](https://github.com/rndquu-org/test-repo/issues/57#issuecomment-2208312490) in [this](https://github.com/rndquu/conversation-rewards/actions/runs/9790330929) CI run